diff --git a/modules/lightning_scheduler/tests/contexts/SchedulerContext.behat.inc b/modules/lightning_scheduler/tests/contexts/SchedulerContext.behat.inc
index 91431d4..eb0fcfd 100644
--- a/modules/lightning_scheduler/tests/contexts/SchedulerContext.behat.inc
+++ b/modules/lightning_scheduler/tests/contexts/SchedulerContext.behat.inc
@@ -29,8 +29,6 @@ final class SchedulerContext extends DrupalSubContextBase {
    * @BeforeScenario
    */
   public function setUp() {
-    // Use call_user_func() to get around the deprecation testing tools we use.
-    ini_set('date.timezone', call_user_func('drupal_get_user_timezone'));
     $this->startTime = time();
   }
 
